cmd/kyverno/main.go

```diff
@@ -606,7 +606,7 @@ func main() {
 	go configurationController.Run(signalCtx, configcontroller.Workers)
 	go eventGenerator.Run(signalCtx, 3)
 
-	if serverIP != "" {
+	if serverIP == "" {
```

Comment:
Why starting the monitor if we are running in debug mode?

Reply:
If serverIP is empty we are NOT in debug mode

Reply:
I was under the impression that , if serverIP is empty, means kyverno running as binary not as container.

Reply:
```sh
flag.StringVar(&serverIP, "serverIP", "", "IP address where Kyverno controller runs. Only required if out-of-cluster.")
```

Reply:
Yes, if serverIP is empty we are running in cluster. Therefore the above line is correct.

Reply:
Ops my bad :(